go/signedexchange/signer.go

```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/WICG/webpackage/go/signedexchange/cbor"
 )
 
+// The "context string" in Step 7.2 of
+// https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#signature-validity
+const ContextString = "HTTP Exchange 1"
```

Comment:
Following the convention of Go, comments should start with `ContextString is...`.

Reply:
Do we need to export this const? If not, this should be `contextString`

Reply:
Done